rules/php-74/src/Rector/LNumber/AddLiteralSeparatorToNumberRector.php

```diff
@@ -76,6 +77,12 @@ public function refactor(Node $node): ?Node
         }
 
         $numericValueAsString = (string) $node->value;
+
+        // ...
+        dump($node->getAttribute(AttributeKey::ORIGINAL_NODE));
+        dump($numericValueAsString);
+        die;
```

Comment:
No way to get original code value :( 

See https://github.com/nikic/PHP-Parser/pull/655